src/components/ble/NotificationManager.h

```diff
@@ -70,7 +69,7 @@ namespace Pinetime {
       size_t beginIdx = TotalNbNotifications - 1; // index of the newest notification
       size_t size = 0;                            // number of valid notifications in buffer
 
-      std::atomic<bool> newNotification {false};
+      bool newNotification;
```

Comment:
I think it would be better to explicitly initialize this to false as you did with the other variables.